crates/rpc/rpc/src/otterscan.rs

```diff
@@ -45,7 +47,22 @@ where
 
     /// Handler for `ots_getTransactionError`
     async fn get_transaction_error(&self, _tx_hash: TxHash) -> RpcResult<String> {
```

Comment:
I think we want to make two changes here: 
* change this to `Option<Bytes>`
* only return Some if Revert

ref:

https://github.com/ledgerwatch/erigon/blob/aefb97b07d1c4fd32a66097a24eddd8f6ccacae0/core/state_transition.go#L130-L130

Reply:
Done.